audio-worklet/test/bitcrusher-test.html

```diff
@@ -14,7 +14,7 @@
 -->
 <!DOCTYPE html>
```

Comment:
The custom build does not support OfflineAudioContext yet, so I think you can remove this test file out of this PR.